src/config.rs

```diff
@@ -33,27 +33,33 @@ pub struct Config {
     pub single_port: bool,
     /// Refuse all write requests, making the server read-only. (default: false)
     pub read_only: bool,
-    /// Duplicate all packets sent from the server. (default: 1)
+    /// Duplicate all packets sent from the server. (default: 0)
```

Comment:
Do this a separate merge request. (You all ready made a separate commit.)

Reply:
https://github.com/altugbakan/rs-tftpd/pull/20